packages/rocketchat-api/server/v1/im.js

```diff
@@ -305,12 +305,16 @@ RocketChat.API.v1.addRoute(['dm.list.everyone', 'im.list.everyone'], { authRequi
 		const { sort, fields, query } = this.parseJsonQuery();
 
 		const ourQuery = Object.assign({}, query, { t: 'd' });
+		const fieldsWithUsernames = () => {
```

Comment:
any reason to have a function to do this instead of just running `delete fields.usernames`?